tests/integration/helpers/format-time-test.ts

```diff
@@ -1,17 +1,14 @@
-import {
-  render,
-  type TestContext as BaseTestContext,
-} from '@ember/test-helpers';
+import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import type IntlService from 'ember-intl/services/intl';
+import type { TestContext as BaseTestContext } from 'ember-intl/test-support';
```

Comment:
The TestContext provided by this addon is the standard test context interface, along with `intl: IntlService` added.